h2o-k8s/tests/clustering/docker-init-script.sh

```diff
@@ -6,6 +6,12 @@ if [[ $string == *"@"* ]]; then
   echo "H2O base path contains at sign. Unable to create K3S cluster."
   exit 1
 fi
+
+# Force in-house version of the image in tests
+sed -i.bu 's,everpeace,harbor.h2o.ai/opsh2oai,' $H2O_BASE/h2o-helm/templates/tests/test-connection.yaml
+rm $H2O_BASE/h2o-helm/templates/tests/test-connection.yaml.bu
```

Comment:
I'm not sure if it's wise to modify part of source - it can get annoying when running the test locally.
I would suggest to undo the change afterwards, e.g.
```suggestion
sed -i.bu 's,everpeace,harbor.h2o.ai/opsh2oai,' $H2O_BASE/h2o-helm/templates/tests/test-connection.yaml
trap 'rm $H2O_BASE/h2o-helm/templates/tests/test-connection.yaml; mv $H2O_BASE/h2o-helm/templates/tests/test-connection.yaml{.bu,}' EXIT
```

Or if it is intended to be ran only in a docker container (linux with GNU coreutils) it could be simplified to:
```suggestion
sed -i 's,everpeace,harbor.h2o.ai/opsh2oai,' $H2O_BASE/h2o-helm/templates/tests/test-connection.yaml
```
(Which has the added "benefit" of failing on macOS with BSD coreutils.)

Reply:
good point, running this locally really could be an issue - I will try to implement your suggestion